environment.yml

```diff
@@ -18,4 +18,5 @@ dependencies:
   - pip:
     - "git+https://github.com/inducer/cgen"
     - "git+https://github.com/inducer/codepy"
+    - "git+https://github.com/opesci/pyrevolve"
```

Comment:
I just realised that Cython itself takes ages to install via pip. Can you please add `cython` to the conda part of our env to speed up the travis install? Thanks

Reply:
I have fixed it but please confirm that this is what you were talking about.

Reply:
:+1: